### PR TITLE
v7.0.0-alpha.7

### PR DESCRIPTION
**Pull request by flaviendelangle**
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,145 @@
...
+  ```
+
+- 🌍 Improve Czech (cs-CZ) locale on the Data Grid
+- 🐞 Bugfixes
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.0.0-alpha.7`
+
+- [DataGrid] Don't evaluate `hasEval` when `disableEval is set (#11516) @reihwald
```
Comment (MBilalShafi) [inline CHANGELOG.md:46]:
```suggestion
- [DataGrid] Don't evaluate `hasEval` when `disableEval` is set (#11516) @reihwald
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,145 @@
...
+- [DataGrid] Replace `eval` with `new Function` (#11557) @oliviertassinari
+- [DataGrid] Warn devs when `autoPageSize` is used with `autoheight` (#11554) @Sboonny
+- [l10n] Improve Czech (cs-CZ) locale (@fdebef) (#11526) @LukasTy
```
Comment (MBilalShafi) [inline CHANGELOG.md:49]:
```suggestion
- [l10n] Improve Czech (cs-CZ) locale (#11526) @fdebef
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,145 @@
...
+- [TreeView] Introduce the `items` prop (#11059) @flaviendelangle
+
+### Docs
+
+- [docs] Add example for TreeView `onNodeExpansionToggle` prop (#11547) @flaviendelangle
+- [docs] Clarify Pickers usage with Luxon (#11545) @LukasTy
+- [docs] Complete transition to next branch (#11521) @oliviertassinari
+- [docs] Fix 404 links in the docs @oliviertassinari
+- [docs] Fix over page fetching @oliviertassinari
+- [docs] Lint next.config.js (#11514) @oliviertassinari
```
Comment (MBilalShafi) [inline CHANGELOG.md:133]:
```suggestion
- [docs] Lint `next.config.js` (#11514) @oliviertassinari
```

**Review by MBilalShafi — APPROVED** [review]
Nice update on the Tree View 👍

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,145 @@
...
+- 🌍 Improve Czech (cs-CZ) locale on the Data Grid
+- 🐞 Bugfixes
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.0.0-alpha.7`
+
+- [DataGrid] Don't evaluate `hasEval` when `disableEval is set (#11516) @reihwald
+- [DataGrid] Replace `eval` with `new Function` (#11557) @oliviertassinari
+- [DataGrid] Warn devs when `autoPageSize` is used with `autoheight` (#11554) @Sboonny
```
Comment (LukasTy) [inline CHANGELOG.md:48]:
```suggestion
- [DataGrid] Warn devs when `autoPageSize` is used with `autoHeight` (#11554) @Sboonny
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,145 @@
...
+- [charts] Fix `null` in line chart using dataset (#11550) @alexfauquette
+
+### Tree View
+
+#### Breaking changes
+
+- The expansion props have been renamed to better describe their behaviors:
+
+  | Old name          | New name                |
+      | :---------------- | :---------------------- |
```
Comment (LukasTy) [inline CHANGELOG.md:81]: Nitpick: Is there a reason for this line to be misaligned with the header line? 🤔

File: packages/x-tree-view/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-tree-view",
-  "version": "7.0.0-alpha.1",
+  "version": "7.0.0-alpha.7",
```
Comment (LukasTy) [inline packages/x-tree-view/package.json:3]: 💯 🙌 🚀

**Review by LukasTy — APPROVED** [review]
Nitpick: Could make sense to include v6 changelog before merging this one. 😉

File: CHANGELOG.md (hunk 5 above)
Comment (flaviendelangle) [reply CHANGELOG.md]: Prettier does not complain when I align it correctly
Fixed :+1:

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,145 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.7
+<!-- generated comparing v7.0.0-alpha.6..next -->
```
Comment (alexfauquette) [inline CHANGELOG.md:7]:
```suggestion

```

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,145 @@
...
+- 🌍 Improve Czech (cs-CZ) locale on the Data Grid
+- 🐞 Bugfixes
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.0.0-alpha.7`
+
+- [DataGrid] Don't evaluate `hasEval` when `disableEval` is set (#11516) @reihwald
+- [DataGrid] Replace `eval` with `new Function` (#11557) @oliviertassinari
+- [DataGrid] Warn devs when `autoPageSize` is used with `autoHeight` (#11554) @Sboonny
```
Comment (romgrk) [inline CHANGELOG.md:48]: #11585 could be included here also, it's a follow-up of that

**Pull request merged**